### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by netoax**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+(Thanks for sending a pull request! Please fill out the blanks below.)
+
+What does this implement/fix? Explain your changes.
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `.` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

**Comment by joaoaneto** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/knot-babeltower/knot-babeltower/pullRequest?prid=4576380&bid=15377301)

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+---------------------------------------------------
+...
+
+Does this close any currently open issues?
+------------------------------------------
+(Use the special github keywords to reference issues: https://help.github.com/en/articles/closing-issues-using-keywords)
+
+...
+
+Any relevant logs, error output, etc?
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+(Thanks for sending a pull request! Please fill out the blanks below.)
+
+What does this implement/fix? Explain your changes.
+---------------------------------------------------
+...
+
+Does this close any currently open issues?
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+------------------------------------------
+(Use the special github keywords to reference issues: https://help.github.com/en/articles/closing-issues-using-keywords)
+
+...
+
+Any relevant logs, error output, etc?
+-------------------------------------
+(If it’s long, please paste to https://pastebin.com/ and insert the link here.)
+
+Any other comments?
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+(Thanks for sending a pull request! Please fill out the blanks below.)
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+What does this implement/fix? Explain your changes.
+---------------------------------------------------
+...
+
+Does this close any currently open issues?
+------------------------------------------
+(Use the special github keywords to reference issues: https://help.github.com/en/articles/closing-issues-using-keywords)
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+...
+
+Any relevant logs, error output, etc?
+-------------------------------------
+(If it’s long, please paste to https://pastebin.com/ and insert the link here.)
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+Any other comments?
+-------------------
+...
+
+Where has this been tested?
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

**Comment by joaoaneto** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/knot-babeltower/knot-babeltower/pullRequest?prid=4576380&bid=15377301)

File: .github/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+---------------------------------------------------
+...
+
+Does this close any currently open issues\?
+------------------------------------------
+(Use the special github keywords to reference issues: <https://help.github.com/en/articles/closing-issues-using-keywords>)
+
+...
+
+Any relevant logs, error output, etc\?
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+-------------------------------------
+(If it’s long, please paste to <https://pastebin.com/> and insert the link here.)
+
+Any other comments\?
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+-------------------
+...
+
+Where has this been tested\?
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 12]
```diff
@@ -0,0 +1,30 @@
+(Thanks for sending a pull request. Please fill out the blanks below.)
+
+What does this implement/fix? Explain your changes
+---------------------------------------------------
+...
+
+Does this close any currently open issues\?
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-heading-punctuation] Don’t add a trailing `?` to headings](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

File: .github/pull_request_template.md [hunk 13]
```diff
@@ -0,0 +1,30 @@
+(Thanks for sending a pull request. Please fill out the blanks below.)
```
Comment (joaoaneto) [inline .github/pull_request_template.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)

**Review by ramonhpr — APPROVED** [review]
LGTM

**Pull request merged**